operation/fhir-operation-member-match/src/main/java/com/ibm/fhir/operation/davinci/hrex/provider/strategy/DefaultMemberMatchStrategy.java

```diff
@@ -202,6 +202,8 @@ public MemberMatchResult executeMemberMatch() throws FHIROperationException {
                         .responseType(ResponseType.NO_MATCH)
                         .build();
             }
+        } catch (FHIROperationException foe){
+            throw foe;
         } catch (Exception e) {
             LOG.throwing(getClass().getSimpleName(), "executeMemberMatch", e);
             throw FHIROperationUtil.buildExceptionWithIssue("Error executing the MemberMatch", IssueType.EXCEPTION);
```

Comment:
you sure we don't want to include the caused by in this case too?  worried about leaking too much info to the client in the response?

Reply:
Yeah, that's why.  I'm intentionally keeping it light.